<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="Welcome">
   <property name="title">
    <string/>
   </property>
   <property name="subTitle">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>665</width>
      <height>394</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Coquery 0.9&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;The corpus query tool&lt;/span&gt;&lt;/p&gt;&lt;p&gt;(c) 2015 Gero Kunter&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;a href=&quot;https://bitbucket.org/gkunter/coquery&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;BitBucket page&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Visit &lt;a href=&quot;https://twitter.com/CoqueryTool&quot;&gt;@CoqueryTool&lt;/a&gt; on Twitter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Logo">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../.designer/backup/logo.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWizardPage" name="CorpusSelection">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>750</width>
      <height>510</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>650</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select corpus:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_corpus"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>650</width>
       <height>170</height>
      </rect>
     </property>
     <property name="title">
      <string>Select query mode</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>47</y>
        <width>169</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radio_mode_context">
         <property name="text">
          <string>Context (KWIC)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_mode_frequency">
         <property name="text">
          <string>Frequency</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_mode_statistics">
         <property name="text">
          <string>Statistics</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="InputSelection">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select the source of the search queries:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="QueryStringBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>710</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="edit_query_string">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>631</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>23</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter query string:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_query_string">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>25</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="button_file_options">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>170</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Options...</string>
     </property>
     <property name="shortcut">
      <string>Alt+O</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>113</y>
       <width>139</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Read queries from file...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_query_file">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>25</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_browse_file">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>133</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
     <property name="shortcut">
      <string>Alt+B</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_file_name">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>551</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="query_options">
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>770</width>
      <height>530</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>39</y>
       <width>750</width>
       <height>471</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>729</width>
        <height>658</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QGridLayout" name="selection_layout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="1,0">
         <property name="horizontalSpacing">
          <number>-1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="coquery_label">
           <property name="text">
            <string>Coquery features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="word_label">
           <property name="text">
            <string>Word features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <layout class="QVBoxLayout" name="speaker_layout">
           <item>
            <widget class="QCheckBox" name="speaker_data_id">
             <property name="text">
              <string>Identifier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="speaker_data_age">
             <property name="text">
              <string>Age</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="speaker_data_sex">
             <property name="text">
              <string>Sex</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="source_label">
           <property name="text">
            <string>Source features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="file_label">
           <property name="text">
            <string>File features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="speaker_label">
           <property name="text">
            <string>Speaker features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QVBoxLayout" name="source_layout">
           <item>
            <widget class="QCheckBox" name="source_id">
             <property name="text">
              <string>Identifier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="source_data_year">
             <property name="text">
              <string>Year</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="source_data_genre">
             <property name="text">
              <string>Genre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="source_data_title">
             <property name="text">
              <string>Title</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="17" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="12" column="1">
          <layout class="QVBoxLayout" name="time_layout">
           <item>
            <widget class="QCheckBox" name="time_data_dur">
             <property name="text">
              <string>Duration</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="time_data_start">
             <property name="text">
              <string>Starting time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="time_data_end">
             <property name="text">
              <string>End time</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <layout class="QVBoxLayout" name="lemma_layout">
           <item>
            <widget class="QCheckBox" name="lemma_data_orth">
             <property name="text">
              <string>Orthographic form</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="lemma_data_phon">
             <property name="text">
              <string>Phonological form</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="lemma_data_pos">
             <property name="text">
              <string>Part-of-speech</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="word_layout" stretch="0,0,0">
           <item>
            <widget class="QCheckBox" name="word_data_orth">
             <property name="text">
              <string>Orthographic form</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="word_data_phon">
             <property name="text">
              <string>Phonological form</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="word_data_pos">
             <property name="text">
              <string>Part-of-speech</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="context_label">
           <property name="text">
            <string>Context features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lemma_label">
           <property name="text">
            <string>Lemma features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="time_label">
           <property name="text">
            <string>Time features</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="Line" name="file_separator">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <layout class="QVBoxLayout" name="context_layout">
           <item>
            <layout class="QHBoxLayout" name="context_left_span_layout">
             <item>
              <widget class="QSpinBox" name="context_left_span"/>
             </item>
             <item>
              <widget class="QLabel" name="context_left_span_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>words in left context</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="context_right_span_layout">
             <item>
              <widget class="QSpinBox" name="context_right_span"/>
             </item>
             <item>
              <widget class="QLabel" name="context_right_span_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>words in right context</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="context_words_as_columns">
             <property name="text">
              <string>Words as columns</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <widget class="Line" name="source_separator">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="Line" name="speaker_separator">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="Line" name="lemma_separator">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="Line" name="word_separator">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <layout class="QVBoxLayout" name="file_layout">
           <item>
            <widget class="QCheckBox" name="file_data_name">
             <property name="text">
              <string>File name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="file_data_path">
             <property name="text">
              <string>File path</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="15" column="0">
          <widget class="Line" name="time_separator">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="file_layout_2">
           <item>
            <widget class="QCheckBox" name="coquery_query_string">
             <property name="text">
              <string>Query string</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="coquery_parameters">
             <property name="text">
              <string>Search parameters</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>750</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
